README.md

```diff
@@ -63,10 +63,10 @@ The following frameworks are available AWS Deep Learning Containers with the dee
 
 | Framework | Version |
 | --- | --- |
-| [TensorFlow](docs/tensorflow.md) | 1.15, 2.1.0, 2.2.0, 2.3.0, 2.3.1 |
-| [MXNet](docs/mxnet.md) | 1.6, 1.7 |
-| [PyTorch](docs/pytorch.md) | 1.4, 1.5, 1.6 |
-| [XGBoost](docs/xgboost.md) | 0.90-2, 1.0-1 ([As a built-in algorithm](docs/xgboost.md#use-xgboost-as-a-built-in-algorithm))|
```

Comment:
Smdebug is supported on the latest versions of all available DLCs.

See [page](https://github.com/aws/deep-learning-containers/blob/master/available_images.md).

Reply:
incorporated